Sources/TSCUtility/Triple.swift

```diff
@@ -120,10 +134,15 @@ public struct Triple: Encodable, Equatable {
     }
 
     fileprivate static func parseABI(_ string: String) -> ABI? {
-        if string.hasPrefix(ABI.android.rawValue) {
+        if string.hasPrefix("android") {
             return ABI.android
         }
-        return nil
+
+        if let version = string.firstIndex(where: { $0.isNumber }) {
```

Comment:
is this logic covered by any of the tests?

Reply:
maybe https://github.com/apple/swift-tools-support-core/pull/330/files#diff-fad259eadf6881f03bfc282e621a5c6a1ab36254e2a4286982b14ac8f9f9bc68R34?

Reply:
and it was broken of course :) Should have used `..<`